buildutils/src/ensure-package.ts

```diff
@@ -216,6 +216,12 @@ export async function ensurePackage(
   // when a package is actually being published.
   delete data.gitHead;
 
+  // Ensure there is a minimal prepublishOnly script
+  if (!data.private && !data.scripts.prepublishOnly) {
+    messages.push(`prepublishOnly script missing in ${pkgPath}`);
+    data.scripts.prepublishOnly = 'npm run build';
```

Comment:
It doesn't really matter since it's just a task-runner here, but I wonder if we should standardize on `jlpm` where possible. I think that's reasonable for a follow-up, however.

Reply:
yes, services is the only package that uses `jlpm` instead of `npm`. Since everything else used `npm`, I just used `npm` here as well.